app/src/debug/java/org/simple/clinic/DebugClinicApp.kt

```diff
@@ -67,8 +68,14 @@ class DebugClinicApp : ClinicApp() {
 
   private fun setupFlipper() {
     with(AndroidFlipperClient.getInstance(this)) {
-      addPlugin(InspectorFlipperPlugin(this@DebugClinicApp, DescriptorMapping.withDefaults()))
+      val context = this@DebugClinicApp
```

Comment:
Maybe move this out of the `with` block and remove the `@DebugClinicApp`?

Reply:
Looks better, will make this change.